src/manifest.py

```diff
@@ -4,13 +4,14 @@
 COMPANY_NIP = "123-456-78-90"
 COMPANY_BANK_ACCOUNT = "12 3456 7890 0000 0000 1234 5678"
 
+# name should be in female form
```

Comment:
This comment is not necessary and will probably be confusing outside the context of this commit

Reply:
it's right above the subscriptions, the subscriptions name ought to be in female form

Reply:
Yes, and it is in female form now. What is the use of this comment after the change?
There isn't a "#name should be in male form" comment above packets, and this comment is also unnecessary.

Reply:
FIXED COMMENT

Reply:
I wanted the comment to be removed, but I'm ok with this change too